lua/lspconfig/server_configurations/buck2.lua

```diff
@@ -0,0 +1,27 @@
+local util = require 'lspconfig.util'
+
```

Comment:
missing window support here . check contribute.md or see what is the other server config do .